.github/workflows/ci.yml

```diff
@@ -1,16 +1,18 @@
 name: CI
-on: [push]
+on:
+  push:
+    branches: [main]
```

Comment:
```suggestion
```

Why only run CI on `main`?

Reply:
It's not innovation :) https://github.com/CheckerNetwork/piece-indexer/pull/8

Reply:
At least for PRs we have already agreed on running CI on all of them

Reply:
I see, that makes sense now.

I updated the triggers to match https://github.com/CheckerNetwork/piece-indexer/pull/8